src/Controller/Controller.php

```diff
@@ -207,7 +207,18 @@
 
         if ($this->isGranted('IS_AUTHENTICATED_REMEMBERED')) {
             $user = $this->get('security.token_storage')->getToken()->getUser();
-            $profile = $this->get('elife.api_sdk.profiles')->get($user->getUsername());
+            $profile = $this->get('elife.api_sdk.profiles')->get($user->getUsername())
+                ->otherwise(function ($reason) use ($user, $request) {
+                    $request->getSession()->invalidate();
+
+                    $response = new RedirectResponse($request->getUri());
+                    $response->headers->clearCookie($this->container->getParameter('session_name'));
```

Comment:
The test passed without this, but real browsers just got stuck in a redirect loop.